contracts/lido_terra_hub/src/contract.rs

```diff
@@ -192,6 +171,80 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> S
     }
 }
 
+pub fn execute_claim_airdrops(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    token: String,
+    stage: u8,
+    amount: Uint128,
+    proof: Vec<String>,
+) -> StdResult<Response> {
+    let params: Parameters = PARAMETERS.load(deps.storage)?;
+    if params.paused.unwrap_or(false) {
```

Comment:
What is the benifit of using `Option<bool>` here instead of just `bool`?
See, `bool` provides two useful values: `true` — contract is paused, `false` — contract is not paused.
On the other hand, `Option<bool>` is strange: `Some(true)` — contract is paused, `Some(false)` and `None` — contract is not paused.
That's really weird. Can we have just `bool` here or is there some requirement to use `Option<bool>`?

Reply:
good point. i cant be sure, but i think it was easier to migrate from parameters struct with no paused field to the struct with the field. not sure rewriting the struct is in scope of the task and may be i'm missing something and we no need to refactor @pr0n00gler

Reply:
yes, this option<bool> was required for a smooth migration. fixing it in this PR is out of scope; @foxpy you can add an issue to the backlog (something like "Fix Migration-related artefacts in terra contracts")